runhouse/resources/hardware/cluster.py

```diff
@@ -271,6 +274,66 @@ def delete_configs(self, delete_creds: bool = False):
 
         super().delete_configs()
 
+    def _setup_creds(self, ssh_creds: Union[Dict, "Secret"]):
```

Comment:
what about supporting a string here representing the rns address of an existing ssh creds resource?

Reply:
I think that makes sense, will add support for it